packages/auth/src/__tests__/AuthProvider.test.tsx

```diff
@@ -15,7 +15,11 @@ type HasRoleAuthClient = AuthClient & {
   hasRole: (role?: string | string[]) => Promise<boolean | null>
 }
 
-let CURRENT_USER_DATA: { name: string; email: string; roles?: string[] } = {
+let CURRENT_USER_DATA: {
+  name: string
+  email: string
+  roles?: string[] | string
```

Comment:
```suggestion
  roles?: Array<string> | string
```
Changed to match `CurrentUser` in AuthProvider

Reply:
equivalent no?

Reply:
100%, so why not do the same in both places?
Personally, for whatever reason, I always do `string[]`, but I prefer `Record<string, string>`. Don't know why I like the utility helper for one of them but not the other...